index.js

```diff
@@ -1,5 +1,78 @@
-console.log("hola");
 
+"use strict";
 const mainContainer = document.querySelector("#app");
+const listOfCards = document.querySelector("#lista");
+const shuffle = document.querySelector(".shuffle");
+const topCard = document.querySelector("#top-card");
+
+const cardsArr = [];
+const clubs = ["clubs", "diamonds", "hearts", "spades"];
+const cards = ["As", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K"];
+
+//GENERANDO ARRAY CON CARTAS
+for (let i = 0; i < 4; i++) {
+  for (let j = 0; j < 13; j++) {
+    let cardObj = {
```

Comment:
This variable doesn't change so you can make it a `const`
```suggestion
    const cardObj = {
```